frontend/src/components/Sidebar.js

```diff
@@ -0,0 +1,97 @@
+import { NavLink } from "react-router-dom";
+import Wrapper from "../styles/Sidebar";
+import {
+  HistoryIcon,
+  HomeIcon,
+  LibIcon,
+  LikeIcon,
+  SubIcon,
+  TrendingIcon,
+  VidIcon,
+} from "./Icons";
+import SidebarAuth from "./SidebarAuth";
+
+function Sidebar({ isSidebarOpen }) {
+  return (
+    <Wrapper open={isSidebarOpen}>
+      <NavLink
+        end
+        to="/"
+        className={({ isActive }) => (isActive ? "active" : "")}
+      >
+        <div className="icon">
+          <HomeIcon />
+          <span>Home</span>
+        </div>
+      </NavLink>
+
+      <NavLink
+        to="/feed/trending"
+        className={({ isActive }) => (isActive ? "active" : "")}
+      >
+        <div className="icon">
+          <TrendingIcon />
+          <span>Trending</span>
+        </div>
+      </NavLink>
+
+      <NavLink
+        to="/feed/playlist"
+        className={({ isActive }) => (isActive ? "active" : "")}
+      >
+        <div className="icon">
+          <SubIcon />
+          <span>Playlist</span>
+        </div>
+      </NavLink>
+
+      <div className="divider"></div>
```

Comment:
Just a suggestion, you can also create an array and map through it for the sidebar component as the code is getting repeated